scripts/Packages/Package_DFN_QFN/size_definitions/qfn_texas.yaml

```diff
@@ -310,3 +310,55 @@ Texas_S-PVQFN-N64_EP:
   #pad_length_addition: 0.5
   #suffix: '_Pad{pad_x:.2f}x{pad_y:.2f}mm_HandSolder'
   #include_suffix_in_3dpath: 'False'
+
+Texas_VQFN_RGE0024H:
+  device_type: 'QFN'
+  manufacturer: 'Texas'
+  #part_number: 'mpn'
+  size_source: 'http://www.ti.com/lit/ds/symlink/tlc5971.pdf#page=42&zoom=200,-13,779'
+  ipc_class: 'qfn' # 'qfn_pull_back'
+  #ipc_density: 'least' #overwrite global value for this device.
+  custom_name_format: 'Texas_RGE0024H{vias:s}'
+  body_size_x_min: 3.9
+  body_size_x: 4.0
+  body_size_x_max: 4.1
+  body_size_y_min: 3.9
+  body_size_y: 4.0
+  body_size_y_max: 4.1
+  lead_width_min: 0.18
+  lead_width_max: 0.3
+  lead_len_min: 0.28
+  lead_len_max: 0.48
+
+  EP_size_x_min: 2.6
+  EP_size_x: 2.7
+  EP_size_x_max: 2.8
+  EP_size_y_min: 2.6
+  EP_size_y: 2.7
+  EP_size_y_max: 2.8
+  EP_paste_coverage: 0.78
```

Comment:
Is there a reason for this much paste coverage? We normally aim for about 65%.
(Especially for the non Thermal vias option. The thermal vias option can have a higher coverage. This helps combat paste loss if the paste overlaps the vias like in this case.)

Reply:
this comes directly from the datasheet...
http://www.ti.com/lit/ds/symlink/tlc5971.pdf#page=45&zoom=140,-151,749
> SOLDER PASTE EXAMPLE
BASED ON 0.125 mm THICK STENCIL
EXPOSED PAD
78% PRINTED COVERAGE BY AREA
SCALE: 20X

i don't know what the 'correct' way here is -
the datasheet also recommends `0.24mm pad width` - but the script calculations generate `0.25mm`

Reply:
The coverage assumes that some solder will be lost to the suggested thermal vias. As i suggested above you can use a lower coverage for the non thermal via option.

Regarding pad width: If you want your part true to the datasheet then this script is the wrong approach. (You need to trust IPC and my implementation of IPC to use this script.)